resolversrv/views.py

```diff
@@ -786,3 +1109,30 @@ def remove():
         payload = dict(request.form)  # post data in form encoding
 
     return DeleteRequest().process_request(payload)
+
+
+@advertise(scopes=['ads:resolver-service'], rate_limit=[1000, 3600 * 24])
+@bp.route('/delete_new', methods=['DELETE'])
```

Comment:
That's fine - you can have `delete` and `delete_new` functions, but there should be only one `/delete` api endpoint; master only needs (and should only know) one set of endpoints. Master treats the other remote services as a blackbox, the API is the contract and the master pipeline shouldn't know/care what is happening behind the curtain. It is going to send data to one set of API endpoints and you can then decide to update table 1.0, 2.0 or both. 

This way, we'll also avoid any breakage/config changes for deployment

Reply:
I have said this several times before, here is one more time, having delete and delete_new is temporary so that I don't break the old endpoint while still would b able to use the new one. eventually, before going to production, after verifying nothing is broken, delete_new would be rename delete, and obviously the old delte would go away.

Reply:
I have noticed you said it, it is not an issue of me not understanding. I do. But I don't see a valid reason to do it that way - to introduce a temporary api, which requires changing master pipeline deployment (or even master pipeline code, if we wanted to use both these endpoints simultaneously).

There is no provision in the master pipeline to be sending data to two endpoints simultaneously (`delete` vs `delete_new`) and there shouldn't be one, resolver is a blackbox as far as master is concerned. And changing master's config just because we want to use one or the other endpoint is much better accomplished by simply making a different release of resolver service.

If these two endpoints are receiving the same data, then it is no-problem (nothing can get broken; just call whatever function you want from inside the api); if they are receiving different data, then it is trivial to route the data - call `old_delete` or `new_delete` functionality inside that public api based on what gets passed.

Reply:
yes Steve was actually working on sending data to the second endpoints, while not touching the first endpoint. Two two endpoint accept different protobuf structures and different set of data.
if everything is clear now, please approve or merge the PR. thanks

Reply:
alright, please rename the endpoints to start with an underscore and `tmp` instead of new; also squash the commits to have  a streamlined history. Then you can merge the PR.